Adapters/AdapterChecklists/RequirementsToTestOrBuildAnAdapter.md

```diff
@@ -15,3 +15,5 @@ access the API:
 tester and the adapter (No shared test instances/accounts/tenants)
 - [ ] The tester must be provided all API keys that are required in order for
 the adapter to interact with the API
+- [ ] Access to the API documentation (in the event that the API documentation is 
+password protected or not otherwise publicly available)
```

Comment:
Maybe we can reformulate it to something like (I was a bit confused when I first read it):

_"...(In case of password protected or not otherwise publicly available API documentation)"_